README.md

```diff
@@ -190,13 +190,19 @@ a specific user:
 sudo ip tuntap add name tap0 mode tap user $USER
 sudo ip link set tap0 up
 sudo ip addr add 192.168.69.100/24 dev tap0
+sudo ip -6 addr add fe80::100/64 dev tap0
+sudo ip -6 addr add fd80::100/64 dev tap0
```

Comment:
Address listed twice (for real this time :smile:)

Reply:
Ugh I'm wrong again! The `fe` -> `fd` keeps confusing me lol

Reply:
Do you think I should use a different prefix to make it clearer? (eg. `fdaa::/64`)

Reply:
Yeah, that would be great! Could you also squash your commits?

Reply:
Done